File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_permission.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusPermission:
    tenant_id: int = None
    organization_id: int = None
    data_scope: str = None
    id: int = None
    uuid: str = None
    created_at: str = None
    updated_at: str = None
    v: int = None
    name: str = None
    code: str = None
    description: str = None
    status: str = None
    sort_order: int = None
    resource_url: str = None
    http_method: str = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusPermission.kt
package com.sdkwork.backend

data class PlusPermission(
    val tenantId: Int? = null,
    val organizationId: Int? = null,
    val dataScope: String? = null,
    val id: Int? = null,
    val uuid: String? = null,
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val v: Int? = null,
    val name: String? = null,
    val code: String? = null,
    val description: String? = null,
    val status: String? = null,
    val sortOrder: Int? = null,
    val resourceUrl: String? = null,
    val httpMethod: String? = null
)
